src/components/WelcomeToSDC.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import {
+    Box,
+    HStack,
+    VStack,
+    Heading,
+    Text,
+    Image,
+    Button,
+    Center,
+    Flex,
+    Grid,
+    GridItem,
+} from "@chakra-ui/react";
+
+export const WelcomeToSDC: React.FC = () => {
+    const title = "Welcome to SDC";
+    const desc1 = "Registration for Summer 2021 classes begins June 31, 2021!";
+    const list1 =
+        "Browse through programs below and then select the day/time preferred! We make sure that 100% of our participants' demonstrated needs are met.";
+    const list2 =
+        "For volunteers, create an account to volunteer for SDC and then select the program you want to volunteer for!";
+    const img1 =
+        "https://user-images.githubusercontent.com/4568003/42373481-4c4b04ac-8115-11e8-97de-d25181aeddfd.png";
+    const img2 =
+        "https://cdn.icon-icons.com/icons2/2248/PNG/512/hand_heart_icon_137523.png";
+    const bigImg =
+        "https://img.webmd.com/dtmcms/live/webmd/consumer_assets/site_images/article_thumbnails/other/active_kids_other/1800x1200_active_kids_other_alt.jpg";
```

Comment:
These images will be replaced right?

Reply:
Can you ask designers for images we should use?

Reply:
added a todo, we can change them later